src/aspire/image/__init__.py

```diff
@@ -78,6 +79,53 @@ def _im_translate2(im, shifts):
     return im_translated
 
 
+def normalize_bg(imgs, bg_radius=1.0, do_ramp=True):
+    """
+    Normalize backgrounds and apply to a stack of images
+
+    :param imgs: A stack of images in L-by-L-by-N array
+    :param bg_radius: Radius cutoff to be considered as background (in image size)
+    :param do_ramp: When it is `True`, fit a ramping background to the data
+            and subtract. Namely perform normalization based on values from each image.
+            Otherwise, a constant background level from all images is used.
+    :return: The modified images
+    """
+    L = imgs.shape[1]
+    grid = grid_2d(L)
+    mask = (grid['r'] > bg_radius)
```

Comment:
I see these 2 lines one after the other a LOT of times throughout our code. Should we just create a `circular_mask(L)` to reduce noise?

Reply:
Sure. Do you prefer to doing it in this PR?

Reply:
`grid_2d` have several options they need to pass to `circular_mask` too.

Reply:
Forget it for now, but think about a good approach here and open an issue if you agree with the problem.

Reply:
The grid_2d have a couple of options to fit the different needs from different modules, Sometime need to shift half pixel and something need to normalize. Ideal the mask should include the same options to fix different needs. I am OK to open an issue to change the code related to this PR.